packages/kbn-lens-embeddable-utils/attribute_builder/visualization_types/layers/xy_data_layer.ts

```diff
@@ -89,11 +86,11 @@ export class XYDataLayer implements ChartLayer<XYDataLayerConfig> {
         ? getHistogramColumn({
             columnName: HISTOGRAM_COLUMN_NAME,
             options: {
-              ...options.buckets.params,
-              sourceField: options.buckets.field ?? dataView.timeFieldName,
               params: {
-                interval: this.layerConfig.options?.params?.interval ?? 'auto',
+                interval: options.buckets.params?.interval ?? 'auto',
```

Comment:
We don't need this here, do we? 

In the `getHistogramColumn` function, there is this line https://github.com/elastic/kibana/commit/992761183f9462732e4506cf67925990197cd15c#diff-8292b9297eb79a1ceab8c2d3096c4273102ba4dfd495e76dc11f06fadb4ef6faR54, that, apparently, will do exactly the same thing, right?

Would this is enough?

```ts
params: {
  ...options.buckets.params,
}
```

Reply:
@crespocarlos, I had to, `interval` shouldn't be optional when we pass `params`
<img width="761" alt="Screenshot 2023-11-09 at 14 03 12" src="https://github.com/elastic/kibana/assets/6838659/9b869c9c-13ad-4e42-972d-11119b8b136b">

Reply:
@fkanout , sorry, I meant to say this:

```ts
options: {
  ...options.buckets.params,
  sourceField: options.buckets.field ?? dataView.timeFieldName,
},
```

Reply:
So that you know – it was updated as agreed on in our meeting. Thanks again @crespocarlos!